cli/cache/mod.rs

```diff
@@ -234,29 +236,41 @@ impl Loader for FetchCacher {
         LoaderCacheSetting::Only => Some(CacheSetting::Only),
       };
       file_fetcher
-        .fetch_with_options(FetchOptions {
-          specifier: &specifier,
-          permissions,
-          maybe_accept: None,
-          maybe_cache_setting: maybe_cache_setting.as_ref(),
-          maybe_checksum: options.maybe_checksum,
+        .fetch_once_with_options(FetchOnceOptions {
```

Comment:
This is the main change. The CLI is now doing a single fetch without following redirects and telling deno_graph about the redirect.

Reply:
`fetch_once` was the existing naming. I can rename it to `fetch_no_redirects`